smib/slack/config.py

```diff
@@ -15,4 +15,4 @@
 
 PLUGINS_DIRECTORY = config('PLUGINS_DIRECTORY', default=ROOT_DIRECTORY / 'slack' / 'plugins', cast=Path)
 
-SPACE_OPEN_ANNOUNCE_CHANNEL_ID = config('SPACE_OPEN_ANNOUNCE_CHANNEL_ID', default='C06UDPLQRP1')
\ No newline at end of file
+SPACE_OPEN_ANNOUNCE_CHANNEL_ID = config('SPACE_OPEN_ANNOUNCE_CHANNEL_ID', default='space-open-announce')
```

Comment:
@sam57719 Is this an example config? In which case it should be set to maybe "ABC123DEF45" or similar to illustrate it's an ID and not a name.

Or have we moved to names instead of IDs for some reason?

Reply:
Can be an ID or the channel name - code is identical. Doesn't matter, but thought `space-open-announce` as the default was better than a random one or an ID specific to one of the slack workspaces

Reply:
I will change it to `SPACE_OPEN_ANNOUNCE_CHANNEL`

Reply:
I need the answer to the question to pass it... Am I right in my guess? I don't want us to move to channel name, we must use ID, and the config item should be called ID, but I'm checking it can take both. If you want to put a channel name in as the default, the fine, but we should encourage ID use and use it ourselves.

Reply:
The slack apis will take a channel ID or channel name in the channel parameter. Either perform exactly the same.